src/main/java/db/migration/V194__Migrate_instructor_userRoles.java

```diff
@@ -0,0 +1,156 @@
+package db.migration;
+
+import org.flywaydb.core.api.migration.jdbc.JdbcMigration;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class V194__Migrate_instructor_userRoles implements JdbcMigration {
+
+    /**
+     * Converts any 'instructor'-ish userRoles into an instructor role with a relevant instructorType
+     *
+     * @param connection
+     * @throws Exception
+     */
+    @Override
+    public void migrate(Connection connection) throws Exception {
+        PreparedStatement psUsers = connection.prepareStatement("SELECT * FROM `Users`");
+
+        try {
```

Comment:
Do not catch exceptions in Java-based migrations

Reply:
Don't use try ... finally without catch. Remove the 'try' and 'finally' altogether.